lib/server.js

```diff
@@ -159,6 +159,11 @@ server.post(
   validateParams,
   comment.createItemComment
 );
+server.put(
```

Comment:
Semantically speaking, this feels more like a `PATCH` request to me

Reply:
Do you think we should keep it flexible and require the consumer of this endpoint to pass in the pin and metadata as well? That way people have the option to update those values if needed. If you prefer to lock it down to just being able to update the comment body, I can update this to PATCH.

Reply:
I don't think so. If we change the pin/metadata you start to change what it is we're actually commenting on. In that case, the old comment should be removed and a new one created. I think we should leave it to just the body

Reply:
Makes sense. I'll push an update to change this to a PATCH in a lil bit.

Reply:
@recursivefunk while it makes sense for it be a `patch`, the project is not using `patch` for any endpoints. It would make sense to keep all the end points consistent.

Reply:
@kepta if other endpoints are using `post` where they should be using `patch`, it makes more sense to me to move towards correcting them. We'd have to bump the API version because that represents a breaking change, but I don't want us to get in the habit of "staying wrong because it's the easiest path forward". That's a valid path forward for some large, complex changes, but I don't think that applies here.

Reply:
I agree, currently at a lot of places we are using `PUT` which is just a`PATCH` in disguise.

As per offline conversation with @recursivefunk , the best way to make `to-fix` endpoints more semantically in line with REST framework would be to move a `v2` API. 

Next actions would be creating a ticket outlining the problem and solution.

Reply:
Ticket https://github.com/osmlab/to-fix-backend/issues/262